### ISSUE TITLE
docker-squash

### ISSUE THREAD

**Issue by jnlickey** [goldmann/docker-squash#213]
Attempted to use docker-squash to squash an image with 37 layers. This is the output that I got.

```
root@myserver:/my/path# docker-squash -t new-image-name:0.0.5 old-image-name:0.0.4
2022-09-21 13:29:31,494 root         INFO     docker-squash version 1.0.9, Docker e42327a, API 1.41...
2022-09-21 13:29:31,495 root         INFO     Using v2 image format
2022-09-21 13:29:31,513 root         INFO     Old image has 37 layers
2022-09-21 13:29:31,513 root         INFO     Checking if squashing is necessary...
2022-09-21 13:29:31,513 root         INFO     Attempting to squash last 37 layers...
2022-09-21 13:29:31,513 root         INFO     Saving image sha256:18e069d4ef4811f267367117a780a17dedf91179c94637066549471cc26386b4 to /tmp/docker-squash-yeu3k3hj/old directory...
2022-09-21 13:29:48,366 root         ERROR    module 'docker' has no attribute 'version_info'
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/docker_squash/image.py", line 357, in _save_image
    if docker.version_info[0] < 3:
AttributeError: module 'docker' has no attribute 'version_info'
2022-09-21 13:29:48,366 root         WARNING  An error occured while saving the sha256:18e069d4ef4811f267367117a780a17dedf91179c94637066549471cc26386b4 image, retrying...
2022-09-21 13:29:48,366 root         INFO     Saving image sha256:18e069d4ef4811f267367117a780a17dedf91179c94637066549471cc26386b4 to /tmp/docker-squash-yeu3k3hj/old directory...
2022-09-21 13:30:05,063 root         ERROR    module 'docker' has no attribute 'version_info'
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/docker_squash/image.py", line 357, in _save_image
    if docker.version_info[0] < 3:
AttributeError: module 'docker' has no attribute 'version_info'
2022-09-21 13:30:05,064 root         WARNING  An error occured while saving the sha256:18e069d4ef4811f267367117a780a17dedf91179c94637066549471cc26386b4 image, retrying...
2022-09-21 13:30:05,064 root         INFO     Saving image sha256:18e069d4ef4811f267367117a780a17dedf91179c94637066549471cc26386b4 to /tmp/docker-squash-yeu3k3hj/old directory...
2022-09-21 13:30:21,818 root         ERROR    module 'docker' has no attribute 'version_info'
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/docker_squash/image.py", line 357, in _save_image
    if docker.version_info[0] < 3:
AttributeError: module 'docker' has no attribute 'version_info'
2022-09-21 13:30:21,819 root         WARNING  An error occured while saving the sha256:18e069d4ef4811f267367117a780a17dedf91179c94637066549471cc26386b4 image, retrying...
2022-09-21 13:30:21,819 root         ERROR    Couldn't save sha256:18e069d4ef4811f267367117a780a17dedf91179c94637066549471cc26386b4 image!
2022-09-21 13:30:21,819 root         ERROR    Execution failed, consult logs above. If you think this is our fault, please file an issue: https://github.com/goldmann/docker-squash/issues, thanks!
```

Thank you for looking into this issue.

Jon

**Comment by BTripp1986** [comment]
This is a duplicate of this issue https://github.com/goldmann/docker-squash/issues/212

**Comment by goldmann** [comment]
Should be fixed with #216, thank you!
